common/cmake/install.cmake

```diff
@@ -28,6 +28,12 @@ function(INSTALL_DEVICE_FILES)
     return()
   endif()
 
+  get_target_property(USE_GRAPH_LIMIT ${DEVICE_TYPE} USE_GRAPH_LIMIT)
```

Comment:
The name `USE_GRAPH_LIMIT` is confusing to me. Would `LIMIT_GRAPH_TO_DEVICE` or `OUTPUT_GRAPH_FOR_DEVICE` be better?

Reply:
Done, renamed to `LIMIT_GRAPH_TO_DEVICE`